composer.json

```diff
@@ -109,20 +109,19 @@
         }
     ],
     "scripts": {
-        "lint": "bin/run-all vendor/bin/parallel-lint DIR --exclude DIR/vendor",
-        "phpcs": "bin/run-all vendor/bin/phpcs -ps DIR",
-        "fix": "bin/run-all vendor/bin/phpcbf DIR",
-        "phpunit": "bin/run-all vendor/bin/phpunit DIR",
-        "phpunit-unit": "bin/run-all vendor/bin/phpunit --testsuite unit DIR",
-        "phpunit-integration": "bin/run-all vendor/bin/phpunit --testsuite integration DIR",
+        "lint": "bin/run-template vendor/bin/parallel-lint DIR --",
+        "phpcs": "bin/run-template vendor/bin/phpcs -ps DIR --",
+        "fix": "bin/run-template vendor/bin/phpcbf DIR --",
```

Comment:
Have you tried ECS? :) It's much more advanced and easier to use
https://github.com/symplify/easy-coding-standard

Reply:
Ooh, it can use sniffs from code sniffer, that is interesting.
This mono repo is trying to stick with a code style applied to most MediaWiki PHP code, so if I could use the sniffs from that while also using easy-coding-standard that sounds appealing!

Reply:
Be sure to check `COMMON` set. It's a coding standard on steroids :)